docs/static/ls-ls-http.asciidoc

```diff
@@ -0,0 +1,9 @@
+[[ls-to-ls-http]]
+==== Logstash-to-Logstash using HTTP
+
+You can set up communication between two Logstash machines using the http output to the http input. 
+
+IMPORTANT: In its current iteration, LS-to-LS using HTTP may not solve all use cases currently addressed by the Beats-Lumberjack approach. 
+We are working on some features that will expand capabilities, and make {ls}-to-{ls} with HTTP easier to configure and use. 
```

Comment:
I dont think we need to tell these to users right now.

Reply:
This is phantom content. The changes you committed actually removed it.  It was still showing up because of the file restructuring I did (without deleting that file).